src/server/locale/US/CONTEXTUAL_COMPONENTS/BUTTONS/mutations/purchase_protection.js

```diff
@@ -0,0 +1,144 @@
+import Logo from '../../../../../message/logos';
+import {
+    textWrap,
+    messageLogoWidth,
+    altNoWrap,
+    setLogoTop,
+    xSmallFallback,
+    hideDisclaimer
+} from '../../../../../message/mediaQueries';
+import { textLogoMutations } from '../../../../../message/logoMutations';
+
+export default {
+    'layout:text': [
+        [
+            'default',
+            ({ textSize }) => ({
+                styles: [
+                    xSmallFallback(textSize * 10.5),
+                    textWrap(textSize * 32, textSize, 'US'),
+                    messageLogoWidth(false, textSize * 4, textSize * 1.25),
+                    setLogoTop(textSize * 10.5),
+                    hideDisclaimer(textSize * 10.5)
+                ],
+                logo: Logo.PP_PAYPAL.COLOR,
+                headline: [
+                    {
+                        tag: 'medium',
+                        br: ['on']
+                    },
+                    {
+                        tag: 'xsmall',
+                        br: ['easier']
+                    }
+                ],
+                disclaimer: ['default']
+            })
+        ],
+        [
+            'logo.type:primary && logo.position:right',
+            ({ textSize }) => ({
+                styles: [
+                    xSmallFallback(textSize * 10.5),
+                    setLogoTop(textSize * 18),
+                    messageLogoWidth(textSize * 6, textSize * 4, textSize * 1.25),
+                    hideDisclaimer(textSize * 10.5),
+                    `.locale--US .message__messaging {padding-right: 0.4rem;}
+                    `
```

Comment:
```suggestion
                    setLogoTop(textSize * 28.5),
                    xSmallFallback(textSize * 10.5),
                    setLogoTop(textSize * 18),
                    messageLogoWidth(textSize * 6, textSize * 4, textSize * 1.25),
                    hideDisclaimer(textSize * 10.5),
                    `.locale--US .message__messaging {padding-right: 0.4rem;}`,
                    `.message__headline .tag--xsmall > span {white-space: normal;}`
```
Adding `setLogoTop` + the `white-space: normal` should help address the issue you were seeing.